README.rst

```diff
@@ -121,6 +121,22 @@ We implement several other checks in order to maintain a consistent code style.
 
 in the repository.
 
+How to cite
+-----------
+
+If you used Adaptive in the preparation of a manuscript, we would appreciate it if you cite us.
```

Comment:
Perhaps `If you used Adaptive in a scientific work, please cite it as follows.`

Reply:
Fixed.